packages/components/accordion/CHANGELOG.md

```diff
@@ -1,230 +1,70 @@
-# @mochi-ui/accordion
```

Comment:
Seem our `CHANGELOG.md` file got changed into one `page-content`, not sure is this your intended

Reply:
Thanks @chinhld12, I think I still got an issue with global replacement by VSCode. Need to manually revert the files